pages/preview.php

```diff
@@ -5,7 +5,13 @@
 use rexfactor\TargetVersion;
 
 $addon = rex_get('addon', 'string');
-echo '<h2>AddOn: '. rex_escape($addon) .'</h2>';
+$addonLabel = $addon;
+
+if ('developer' === $addon) {
+    $addonLabel .= ': modules/templates';
+}
+
+echo '<h2>AddOn: '. rex_escape($addonLabel) .'</h2>';
```

Comment:
mir wäre lieber wenn wir eine methode hätten um das addon label auszugeben, sodass wir die logic nicht 3x im addon wiederholen

Reply:
@staabm da hast Du natürlich vollkommen Recht.
Am besten in der Klasse RexFactor!?

z.B. getAddonLabel($addon)

liefert das Addon und bei Developer + modules/templates.